h2o-r/h2o-package/R/frame.R

```diff
@@ -1934,7 +1934,7 @@ h2o.which <- function(x) {
 #' @param na.rm \code{logical}. Indicate whether missing values should be removed.
 #' @param axis \code{integer}. Indicate whether to calculate the mean down a column (0) or across a row (1).
 #' @return Returns an H2OFrame object.
-#' @seealso \code{\link[base]{which.max}} for the base R method.
+#' @seealso \code{\link[base]{which.min}} for the base R method.
```

Comment:
```suggestion
#' @seealso \code{\link[base]{which.max}} for the base R method.
```

Reply:
@jangorecki Do you know why this was changed from max to min?

Reply:
Manual page of `?which.max` is located under `which.min`. Documentation links should have always link manual pages rather than topics (`which.max` is topic), but it was never tested before. Now it is.

Reply:
@jangorecki Ok yes, you're right.  Sorry I missed that.  I asked an R friend about this topic and this is what she said:

> This has been discussed a lot recently on the r-devel mailing list. Gabor Csardi is, I believe, preparing a patch to R itself that might solve the root problem and allow what many of us want to do (what you’re doing) to continue.
However, because it’s unclear how that will turn out and on what timescale, a workaround has already been put into roxygen2 v7.1.1, which was recently released:
https://roxygen2.r-lib.org/news/index.html#roxygen2-7-1-1-2020-06-27
If you make your docs with roxygen2, you can keep linking to the desired topic (or alias) and roxygen2 will look up the correct page or file when it builds the .Rd files.

So with that in mind, perhaps the least invasive solution (where we can keep the references to the exact functions that we want (even if they are only "topics"), would be to build the documentation with roxygen2 v7.1.1.  What do you think?

Reply:
Building documentation with 7.1.1 didn't solve the problem. Not sure but turning links from Rd format to Markdown could help here, haven't try because it would be better to replace all links then rather than some of them.